lms/djangoapps/courseware/date_summary.py

```diff
@@ -139,7 +140,7 @@ class TodaysDate(DateSummary):
     """
     css_class = 'todays-date'
     is_enabled = True
-    date_format = '%b %d, %Y (%H:%M {utc})'.format(utc=_('UTC'))
+    date_format = u'%b %d, %Y (%H:%M {utc})'.format(utc=ugettext_lazy('UTC'))
```

Comment:
Hmm, not sure the ugettext_lazy is enough here, since the .format() will happen at import time.